fineract-core/src/main/java/org/apache/fineract/accounting/common/AccountingConstants.java

```diff
@@ -303,7 +303,7 @@ public enum AccrualAccountsForSavings {
         ESCHEAT_LIABILITY(14), //
         FEES_RECEIVABLE(15), //
         PENALTIES_RECEIVABLE(16), //
-        INTEREST_PAYABLE(17);
+        INTEREST_PAYABLE(17), INTEREST_RECEIVABLE(18);
```

Comment:
Please follow the code style:
- 1 entry, 1 row and // at the end to avoid reformatting!